### ISSUE TITLE
Lowers Resolution

### ISSUE THREAD

**Issue by falconwingz88** [KwaiVGI/LivePortrait#244]
I'm using a 1080p video as an input, after the conversion i downloaded it, and saw the bitrate and the resolution of the video has been decreased. 
Is there a way to keep the resolution the same as the input, or maybe upres it somehow

Thankyou 
![image](https://github.com/user-attachments/assets/f652cfcc-d2ed-4c64-9607-d78682eb2db6)

**Comment by cleardusk** [comment]
https://github.com/KwaiVGI/LivePortrait/issues/227#issuecomment-2251126859 @falconwingz88 

Another similar acount, just raise an issue and will be deleted?

**Comment by falconwingz88** [comment]
hi, i just saw the #227 you mentioned, then opened the config.
saw this settings, does this contribute to the bitrate / data rate reduction ?
`flag_use_half_precision: bool = True`

**Comment by zzzweakman** [comment]
> saw this settings, does this contribute to the bitrate / data rate reduction ?

Yes, `flag_use_half_precision: bool = True` helps save memory and improve inference speed, provided your GPU supports half-precision inference.

**Comment by thatnerdyaigirl** [comment]
Hey, would the quality be improved further than 1920? Also, when I updated to 1920, the lip movement is not good, after updating smooth variance, there is jitter and still mouth is not opening properly
